library/options.inc.php

```diff
@@ -94,7 +94,7 @@ function generate_select_list($tag_name, $list_id, $currvalue, $title, $empty_na
     if ($multiple) {
         $tag_name_esc = $tag_name_esc . "[]";
     }
-    $s .= "<select name='$tag_name_esc'";
+    $s .= "<select name='$tag_name_esc' class='form-control' style='width: auto; display: inline-block'";
```

Comment:
`form-control` is bootstrap
Everything else is for proper display, because bootstrap elements default to width: 100% and display: inline I believe